Add Styling to the view job file

EP-Wick opened the pull request:
I have created a simple neat styling within the view job files, the web page should have "Detail of the Job" as the title and two separated columns where the job description is exclusively at the right side. In addition, I have also created 2 buttons to apply (accept) or decline the job offer, but this would still need to implemented with backend functions for it to work.

Feel Free to improve or update it in anyway

EP-Wick committed:
“mesage”

EP-Wick committed:
“mesage”

EP-Wick committed:
added jobs and hire file to split main

EP-Wick committed:
“candidates”

EP-Wick committed:
updated my own branch

EP-Wick committed:
“mesage”

EP-Wick committed:
job-detail

EP-Wick committed:
Merge branch 'main' of https://github.com/hernancx09/UWA_CITS_3402_Project into branch-Edward

EP-Wick committed:
View_jobs
I have created styling for the view jobs html file
I have also implemented the styling from the style main css file into the view jobs which adds color, hover, etc

sandoncj commented:
It's a good start. I think it needs to be altered though to seem more consistent with the rest of the apps styling. Also I think it'll look better as one column instead of two

sandoncj commented:
We also only need an apply button since this is a view of the job listing not accepting a candidate for work

EP-Wick committed:
Re-test
This is just another test

EP-Wick committed:
Updated the view jobs file
I have removed the second column and combined it into a singular centre column on top of 'Detail of Job' title
I have successfully centred the job description and changed the button from 'Apply to Job' to just 'Apply'
The styling should still match with the theme of our website, but if it still doesn't please feel free to change it

EP-Wick committed:
Added styling for Viewing Messages
I have created Styling based of similar design to our other webpages,
Added a white border to split into 3 section
Also added a title for users to know what page they are on
Please review it and add any necessary changed if required

EP-Wick committed:
Updating stle for view job
Job descrition is still in progress do not push to main yet

EP-Wick committed:
Updates the view jobs again
The styling for viewing jobs should no longer interfere with the main jobs page
feel free to review it

EP-Wick committed:
updated my branch

EP-Wick committed:
Updated Quick Apply
Fixed the size of the message box as for reasons unknwon, it was minimized

EP-Wick committed:
Added Comments to Template
I have added comments to most of the html file which provides readibility and understanding of what the codes do
Some of the comments within the styling are also there to preovide insight on how the styling are affected

EP-Wick committed:
Pushing

EP-Wick committed:
test commit

testing commit

sandoncj committed:
Merge branch 'main' into branch-Edward

EP-Wick committed:
Removed the status section

I have removed the "Status Section" within the view jobs page, Since it was no longer needed

EP-Wick committed:
Merge branch 'branch-Edward' of https://github.com/hernancx09/UWA_CITS_3402_Project into branch-Edward
